openpype/tools/loader/model.py

```diff
@@ -272,11 +272,13 @@ def setData(self, index, value, role=QtCore.Qt.EditRole):
 
                 # update availability on active site when version changes
                 if self.sync_server.enabled and version_doc:
-                    repre_info = self.sync_server.get_repre_info_for_versions(
-                        project_name,
-                        [version_doc["_id"]],
-                        self.active_site,
-                        self.remote_site
+                    repre_info = list(
+                        self.sync_server.get_repre_info_for_versions(
+                            project_name,
+                            [version_doc["_id"]],
+                            self.active_site,
+                            self.remote_site
+                        )
                     )
```

Comment:
Wouldn't `next` make more sense here since you always just want the first entry?

Reply:
To be honest I don't like using `next` for such long expressions especially when it would make it even longer.
```
repre_info = next(
    self.sync_server.get_repre_info_for_versions(
        project_name,
        [version_doc["_id"]],
        self.active_site,
        self.remote_site
    ),
    None
)
```